src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
+  }
+
+  return days * 40;
```

Comment:
Why you remove variables? You should keep all `magic` numbers in variables

Reply:
Because when I do as written in the previous comments, all my tests fail and I no longer know what to do to close this task :)